clients/admin-ui/src/home/types.ts

```diff
@@ -1,5 +1,6 @@
 export type ModuleCard = {
   backgroundColor: string;
+  dataTestId: string;
```

Comment:
it feels a little weird to me that we have to specify the `dataTestId` here. could whatever renders the card just give it a `data-testid` based off of its name or key instead?